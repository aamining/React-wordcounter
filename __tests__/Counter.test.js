import React from 'react'
import renderer from 'react-test-renderer'
import Counter from '../src/Counter'
describe('A counter', ()=> {
  it ('displays the count', () => {
    const counter = renderer.create(<Counter legend = "count count={34}" />)
    expect(counter.toJson).toMatchSnapshot();

  });
});
