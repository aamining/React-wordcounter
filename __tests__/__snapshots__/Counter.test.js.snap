// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`A counter displays the count 1`] = `undefined`;
